src/custom-elements/src/components.d.ts

```diff
@@ -12,6 +12,87 @@ import '@stencil/core';
 
 export namespace Components {
```

Comment:
This file need not to be pushed to GitHub, It get's generated if someone tries to run the project.